apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -275,6 +275,18 @@ public final class AppConfiguration extends BaseAppConfiguration {
    */
   public static final String SSM_PATH_PROCESS_DLQ = "rda/job/process_dlq";
 
+  /**
+   * The path of the SSM parameter that should be used to determine if the cleanup tasks that remove
+   * old FISS and MCS claims should be run on subsequent job runs.
+   */
+  public static final String SSM_PATH_RUN_CLEANUP = "rda/job/run_cleanup";
+
+  /** The maximum number of claims that will be removed during a cleanup task run. */
+  public static final String SSM_PATH_CLEANUP_RUN_SIZE = "rda/job/cleanup_run_size";
+
+  /** The number of claims to remove as part of a single transaction during a cleanup task run. */
+  public static final String SSM_PATH_CLEANUP_TRANSACTION_SIZE = "rda/job/cleanup_transaction_size";
```

Comment:
It would be nice to group these.  Like `rda/job/cleanup/*`.

Reply:
Done